src/state/queries/post-feed.ts

```diff
@@ -370,14 +375,12 @@ function createApi(
   params: FeedParams,
   feedTuners: FeedTunerFn[],
 ) {
-  if (feedDesc === 'home') {
+  if (feedDesc === 'following') {
```

Comment:
This PR also normalizes how we refer to following. No longer `following` (the og) or `home`. It's just `following`, which receives `home`s treatment.